integration-tests/tests/utils/config.py

```diff
@@ -0,0 +1,17 @@
+from dataclasses import dataclass
+from pathlib import Path
+
+
+@dataclass(frozen=True)
+class BaseConfig:
+    clp_bin_dir: Path
+    clp_package_dir: Path
+    clp_sbin_dir: Path
+    test_output_dir: Path
+    uncompressed_logs_dir: Path
```

Comment:
I would consider calling it logs_dir, or source_logs_dir. having `uncompressed` here is actually quite confusing since it makes me think about decompressed_logs.

Reply:
Also, I believe CLP can ingest archived logs. can we design the flow in a way such that only CLP_S require to extract the log?

Reply:
@haiqi96 The comment regarding leaving ingesting tarball to future